android/src/main/kotlin/com/gojuno/commander/android/Adb.kt

```diff
@@ -39,25 +44,30 @@ fun connectedAdbDevices(): Observable<Set<AdbDevice>> = process(listOf(adb, "dev
 
             shouldRetry
         }
-        .map {
+        .flatMapIterable {
             it
                     .substringAfter("List of devices attached")
                     .split(System.lineSeparator())
                     .map { it.trim() }
                     .filter { it.isNotEmpty() }
                     .filter { it.contains("online") || it.contains("device") }
-                    .map {
+        }
+        .flatMap { line ->
+            deviceModel(line.substringBefore("\t"))
```

Comment:
Hmm, I don't like that we have to do it here, let's move parsing of actual decide model into deviceModel function?